### PR TITLE
UI/CI/rtmp-services: Remove Mixer

### PR DESCRIPTION
**Pull request by derrod**
### Description
Mixer is going to be shut down on July 22nd, this PR removes the Mixer integration, services, and CI variables.

### Motivation and Context
Mixer is dead.

### How Has This Been Tested?
OBS still compiles.

### Types of changes
- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by dodgepong** [comment]
Converting this to draft so it doesn't get accidentally merged too soon.

**Comment by derrod** [comment]
Also a little note: FTL output must remain (for now) as Restream and YouNow are using it.

**Comment by Fenrirthviti** [comment]
Should we reach out to Restream and YouNow if they intend to continue support of FTL now that Mixer is shutting down?

**Comment by derrod** [comment]
Possibly. At least for Restream I imagine they will discontinue support ofr it as well. YouNow OTOH uses it exclusively.

File: UI/data/locale/en-US.ini [hunk 1]
```diff
@@ -833,7 +830,7 @@ Basic.Settings.Audio.PeakMeterType="Peak Meter Type"
...
 Basic.Settings.Audio.PeakMeterType.TruePeak="True Peak (Higher CPU usage)"
 Basic.Settings.Audio.MultiChannelWarning.Enabled="WARNING: Surround sound audio is enabled."
-Basic.Settings.Audio.MultichannelWarning="If streaming, check to see if your streaming service supports both surround sound ingest and surround sound playback. Facebook 360 Live, Mixer RTMP, Smashcast are examples where surround sound is fully supported. Although Facebook Live and YouTube Live both accept surround ingest, Facebook Live downmixes to stereo, and YouTube Live plays only two channels.\n\nOBS audio filters are compatible with surround sound, though VST plugin support isn't guaranteed."
+Basic.Settings.Audio.MultichannelWarning="If streaming, check to see if your streaming service supports both surround sound ingest and surround sound playback. Facebook 360 Live, Smashcast are examples where surround sound is fully supported. Although Facebook Live and YouTube Live both accept surround ingest, Facebook Live downmixes to stereo, and YouTube Live plays only two channels.\n\nOBS audio filters are compatible with surround sound, though VST plugin support isn't guaranteed."
```
Comment (dodgepong) [inline UI/data/locale/en-US.ini:833]: Should probably read "Facebook 360 Live and Smashcast"

Comment (derrod) [reply UI/data/locale/en-US.ini:833]: I'd probably remove the Smashcast mention entirely since that site is basically dead.

Comment (dodgepong) [reply UI/data/locale/en-US.ini:833]: Either way. Both the previous iteration of this sentence and the current iteration are both grammatically incorrect.

**Pull request merged**